src/components/ClosedFilter.vue

```diff
@@ -0,0 +1,128 @@
+<template>
```

Comment:
No need to have a separate filter component for closed page we can conditionally display the filters options on the basis of current page and use a single filter component

Reply:
Rename this file to `Filters` and the name as well.